### PR TITLE
Create page for VS Code release 1.1.0

### PR DESCRIPTION
**Pull request by Shanedell**

### CODE REVIEW

File: site/_vscode/1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+This release contains new features
+
+#### New Features
+
+* Initial omega-edit client.
+    * Enabled via command palette command so that it is hidden by default.
+    * The UI displays a number of panels. The offset, encoded data, decoded data (main editing area). As well as 3 viewports set to specific bytes of the file currently. Some of the dropdowns and areas are not currently interactive but will be in the future.
+    * Actions supports currently:
```
Comment (stevedlawrence) [inline site/_vscode/1.1.0.md:29]: Maybe reword as `Actions currently supported:`.

Also should the actions also be indented a level so they are a sub-list of the "supported actions"?

**Review by stevedlawrence — APPROVED** [review]
+1, minor suggestions

File: site/_vscode/1.1.0.md (hunk 1 above)
Comment (Shanedell) [reply site/_vscode/1.1.0.md]: Yes I missed the tab on this and didn't realize. Should be fixed now.

**Review by tuxji — APPROVED** [review]
+1
